ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__automated-release-stable-4.13-upgrade-from-stable-4.12.yaml

```diff
@@ -0,0 +1,107 @@
+base_images:
+  ansible:
+    name: "4.13"
```

Comment:
Most of the images are used for installation, so the version should align with the "upgrade from" version

Reply:
thanks submitted new change according to this